source/dub/internal/utils.d

```diff
@@ -236,19 +236,22 @@ version (Have_vibe_d_http)
 
 	Note: Timeouts are only implemented when curl is used (DubUseCurl).
```

Comment:
this note should now also say that authentication only works with curl.

See https://github.com/vibe-d/vibe.d/blob/94c19a4e7839eb30a613424bf13606a623079bd4/inet/vibe/inet/urltransfer.d#L34